tests/mr/host/mr_ref_tests.cpp

```diff
@@ -233,14 +233,17 @@ TYPED_TEST(MRRefTest, UnsupportedAlignmentTest)
   for (std::size_t num_trials = 0; num_trials < NUM_TRIALS; ++num_trials) {
     for (std::size_t alignment = MinTestedAlignment; alignment <= MaxTestedAlignment;
          alignment *= TestedAlignmentMultiplier) {
+#ifdef NDEBUG
```

Comment:
Maybe this is related to the CMakeLists.txt change I noted above? What's the rationale for this change?

Reply:
In debug mode, the deallocation will assert, crashing the test.